core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1697,4 +1706,54 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 
 		return $label;
 	}
+
+	/**
+	 * @param array<int|string> $ids
+	 */
+	protected function preloadCurrentRecords(array $ids, string $table): void
```

Comment:
just noticed that this will always override/reload something even if it is always in the cache. Maybe we should add a `$noCache` option to prevent unnecessary reloading?

Reply:
I actually tested. Not a single hit at all in any mode.